examples/megatron/configs/llama2/vllm_policy_inference.yaml

```diff
@@ -29,6 +29,9 @@ max_num_batched_tokens: ${max_num_batched_tokens:32768}
 max_paddings: ${max_paddings:512}
 num_scheduler_steps: ${num_scheduler_steps:1}
 
+# engine config
+enforce_eager: ${enforce_eager:True}
```

Comment:
title should change as well, which shows "default=False"

Reply:
done